include/osquery/core.h

```diff
@@ -92,10 +93,47 @@ osquery::QueryData query(const std::string& q, int& error_return, sqlite3* db);
  * virtual tables attached. This method is the main abstraction for creating
  * SQLite3 databases within osquery.
  *
+ * Note: osquery::initOsquery must be called before calling createDB in order
```

Comment:
Can you clarify what you mean by this comment? I didn't change anything here, just adding the note because it bit me.